Documentation/botr/mixed-mode.md

```diff
@@ -0,0 +1,36 @@
+# Mixed Mode Assemblies
+## Introduction
+Most interoperability between managed and native code uses P/Invokes, COM, or WinRT. Since P/Invokes and COM are bound to native code at runtime, they’re susceptible to mistakes ranging from incorrect naming to subtle mistakes in signatures that cause stack corruption. COM can also be used to call from native to managed code, but it requires a significant amount of setup. WinRT avoids those problems but isn’t available in all cases.
```

Comment:
> Most interoperability between managed and native code uses P/Invokes, COM, or WinRT

Is this statement done in the historical context of .NET Framework? It may be worth to clarify it.

Reply:
I'm not quite sure what you mean here. Is there a context where P/Invokes, COM, and WinRT weren't the majority of interop?

Reply:
In .NET Core today, it is P/Invoke and a bit of COM on Windows, no WinRT.
In Xamarin Android, it is P/Invoke and Android Java bridge, no COM or WinRT.
In Xamarin.iOS, it is P/Invoke and ObjectiveC bridge, no COM or WinRT.